docker/Dockerfile

```diff
@@ -1,11 +1,23 @@
 FROM ubuntu:24.04
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install --yes git build-essential pkg-config libcjson-dev openjdk-17-jdk gradle libbcprov-java libgoogle-gson-java libssl-dev gcc g++ libbotan-2-dev nlohmann-json3-dev python3 python3-pip rustc cargo iputils-ping telnet wget && rm -fr /var/cache/apt/* /var/lib/apt/lists/*
+RUN apt-get update && apt-get install --yes curl wget zip git build-essential pkg-config libcjson-dev openjdk-17-jdk libbcprov-java libgoogle-gson-java libguava-java libssl-dev gcc g++ libbotan-2-dev nlohmann-json3-dev python3 python3-pip rustc cargo iputils-ping telnet wget && rm -fr /var/cache/apt/* /var/lib/apt/lists/*
 RUN pip3 install --break-system-packages cryptography requests
 
 RUN GO_VERSION=$(wget -qO- https://go.dev/VERSION?m=text | head -n 1) && wget https://go.dev/dl/${GO_VERSION}.linux-amd64.tar.gz && rm -rf /usr/local/go && tar -C /usr/local -xzf ${GO_VERSION}.linux-amd64.tar.gz && rm -rf ${GO_VERSION}.linux-amd64.tar.gz
 ENV PATH $PATH:/usr/local/go/bin
 
 COPY rust /rust
 RUN cd /rust && cargo vendor
+
+# Install SDKMAN
+RUN curl -s "https://get.sdkman.io" | bash
```

Comment:
Hier wird einfach ein Skript von einer Seite geladen und das direkt ausgeführt? Was ist SDKMAN? Braucht man das wirklich? Gefällt mir ÜBERHAUPT nicht.

Reply:
SDKMAN ist ein, wie der Name leicht andeutet, Dev-Kit Package Manager und im Falle von Gradle der offizielle Weg für die aktuellste Version. Das "curl into bash" ist aus den [offiziellen Docs](https://sdkman.io/install) entnommen.
Alles was es installiert ist dann in $HOME/.sdkman/ und kann theoretisch nach der Installation von Gradle (wofür ich es ja eig nur brauche) wieder gelöscht werden

Reply:
Ja das hatte ich auch in den offiziellen Docs gelesen, es ist aber halt schon irgendwie scheußlich. Geht Gradle auch ohne SDKMAN oder ist das viel komplizierter?

Reply:
Prinzipiell gehts auch so (Binary direkt runterladen und einfach in /usr/local/bin schmeißen), ich empfands so in dem Moment einfach nur als einfacher